lapack/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/lapack/src)
+LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/lapack/impl)
+LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/lapack/eti)
+LIST(APPEND KK_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/lapack/eti)
```

Comment:
Remove duplicate

Reply:
Thanks, Luc. One is 'CURRENT_SOURCE_DIR', the other is 'CURRENT_BINARY_DIR'. The same happens in blas/CMakeLists.txt. Shouldn't we keep like the way it is now?